framework/src/org/checkerframework/common/value/ValueVisitor.java

```diff
@@ -260,4 +260,33 @@ public Void visitTypeCast(TypeCastTree node, Void p) {
         }
         return super.visitTypeCast(node, p);
     }
+
+    /**
+     * Overridden to issue errors at the appropriate place if an IntRange or ArrayLenRange
+     * annotation has from > to. from > to either indicates a user error when writing an annotation
```

Comment:
The Javadoc isn't building because of `>`.  You can use `&lt;` or `{@code from > to}`.